jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsListViewImpl.java

```diff
@@ -38,6 +42,7 @@
 @Dependent
 @Templated
 public class CaseActionsListViewImpl extends AbstractView<CaseActionsPresenter> implements CaseActionsPresenter.CaseActionsListView {
+    private int PAGE_SIZE = 4;
```

Comment:
@bsremac Sorry I saw this comment after updating the pr. 
I'm going to do those changes and refresh the PR. Thanks!